java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/cp/CPMapTest.java

```diff
@@ -35,16 +45,23 @@ public class CPMapTest extends HazelcastTest {
     public int maps = 1;
     // number of distinct keys to create and use per-map; key domain is [0, keys)
     public int keys = 1;
+    // number of distinct value to create on every client (total number of values is valuesPerClient * clients)
+    public int valuesPerClient = 1;
     // size in bytes for each key's associated value
     public int valueSizeBytes = 100;
+
     private List<CPMap<Integer, String>> mapReferences;
 
-    private String v; // this is always the value associated with any key; exception is remove and delete
+    private String loadGeneratorValues[]; // this is always the value associated with any key; exception is remove and delete
 
+    private IList<CpMapOperationCounter> operationCounterList;
+    private ISet<String> allValues;
```

Comment:
Why is the ISet needed? Every load generator will generate exactly the same set of values because they share the same seed. So what is the purpose of sharing exactly the same values?